### PR TITLE
Add support for Amazon Linux based images

### PR DESCRIPTION
**Pull request by thimslugga**
This is related to #47 and https://github.com/toolbx-images/images/pull/48

### CODE REVIEW

File: amazonlinux/2/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+[Toolbox](https://containertoolbx.org/) is a tool for Linux operating systems, which allows the use of containerized command line environments. It is built on top of [Podman](https://podman.io/) and other standard container technologies from [OCI](https://opencontainers.org/).
```
Comment (travier) [inline amazonlinux/2/README.md:1]: Let's remove this file. We don't include it in the other images.

File: amazonlinux/2/Containerfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
+FROM public.ecr.aws/amazonlinux/amazonlinux:2
+
+ENV NAME=amazonlinux-toolbox VERSION=2
+LABEL com.github.containers.toolbox="true" \
+  name="$NAME" \
```
Comment (travier) [inline amazonlinux/2/Containerfile:5]: We're removing those variables from the image ENV. See https://github.com/toolbx-images/images/issues/50

File: amazonlinux/2/Containerfile [hunk 3]
```diff
@@ -0,0 +1,27 @@
+FROM public.ecr.aws/amazonlinux/amazonlinux:2
+
+ENV NAME=amazonlinux-toolbox VERSION=2
```
Comment (travier) [inline amazonlinux/2/Containerfile:3]: We're removing those variables from the image ENV. See https://github.com/toolbx-images/images/issues/50

File: amazonlinux/2/Containerfile [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+LABEL com.github.containers.toolbox="true" \
+  name="$NAME" \
+  version="$VERSION" \
+  usage="This image is meant to be used with the toolbox command" \
+  summary="Base image for creating Amazon Linux 2 toolbx containers" \
+  maintainer="Adam Kaminski <adam@adamkaminski.com>"
+
+RUN sed -i '/tsflags=nodocs/d' /etc/yum.conf
+
+COPY README.md packages extra-packages missing-docs /
```
Comment (travier) [inline amazonlinux/2/Containerfile:13]:
```suggestion
COPY packages extra-packages missing-docs /
```

File: amazonlinux/2/justfile [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# justfile
```
Comment (travier) [inline amazonlinux/2/justfile:1]: Let's not include this one either

File: amazonlinux/2022/Containerfile [hunk 6]
```diff
@@ -0,0 +1,25 @@
+FROM public.ecr.aws/amazonlinux/amazonlinux:2022
+
+ENV NAME=amazonlinux-toolbox VERSION=2022
```
Comment (travier) [inline amazonlinux/2022/Containerfile:3]: idem

File: amazonlinux/2022/Containerfile [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+LABEL com.github.containers.toolbox="true" \
+  name="$NAME" \
+  version="$VERSION" \
+  usage="This image is meant to be used with the toolbox command" \
+  summary="Base image for creating Amazon Linux toolbx 2022 containers" \
+  maintainer="Adam Kaminski <adam@adamkaminski.com>"
+
+COPY README.md missing-docs extra-packages /
```
Comment (travier) [inline amazonlinux/2022/Containerfile:11]:
```suggestion
COPY missing-docs extra-packages /
```

File: amazonlinux/2022/justfile [hunk 8]
```diff
@@ -0,0 +1,28 @@
+# justfile
```
Comment (travier) [inline amazonlinux/2022/justfile:1]: idem

**Review by travier — CHANGES_REQUESTED** [review]

File: amazonlinux/2/Containerfile (hunk 3 above)
Comment (thimslugga) [reply amazonlinux/2/Containerfile]: I've removed the ENV and replaced with the name and version

**Comment by thimslugga** [comment]
> Thanks. This is also missing GitHub workflow config.

added

**Review by travier — APPROVED** [review]

**Comment by travier** [comment]
Thanks! I'll invite you to the org as a maintainer.

**Comment by travier** [comment]
Well, now you'll have to fix the containerfiles 🙂.

**Pull request closed without merging**